backend/kafka-queue/types.go

```diff
@@ -149,7 +141,6 @@ type Message struct {
 	AddSessionProperties             *AddSessionPropertiesArgs
 	PushBackendPayload               *PushBackendPayloadArgs
 	PushMetrics                      *PushMetricsArgs
-	MarkBackendSetup                 *MarkBackendSetupArgs
```

Comment:
safe to remove this from the message though as the json deserialization will ignore keys not part of this struct